src/Microsoft.SqlTools.ResourceProvider.Core/Discovery/AzureDatabaseDiscoveryProvider.cs

```diff
@@ -121,7 +121,7 @@ public AzureDatabaseDiscoveryProvider()
                     IEnumerable<IAzureUserAccountSubscriptionContext> subscriptions = await GetSubscriptionsAsync(string.IsNullOrEmpty(serverName));
                     if (!cancellationToken.IsCancellationRequested)
                     {                       
-                        result = await AzureUtil.ExecuteGetAzureResourceAsParallel(null, subscriptions, serverName, cancellationToken,
+                        result = await AzureUtil.ExecuteGetAzureResourceAsParallel((object)null, subscriptions, serverName, cancellationToken,
```

Comment:
I thought null would implicitly convert to object?

Reply:
nevermind, it looks like you need the cast for the template to detect the param type.

Reply:
Yep, that's the reason here. I could add in 2 separate util methods in the future, but for now this works OK.